Zend/zend_language_parser.y

```diff
@@ -93,6 +93,7 @@ static YYSIZE_T zend_yytnamerr(char*, const char*);
 %token <ast> T_NAME_RELATIVE "namespace-relative name"
 %token <ast> T_NAME_QUALIFIED "namespaced name"
 %token <ast> T_VARIABLE  "variable"
+%token <ast> T_ATTRIBUTE "attribute"
```

Comment:
This one should not have `<ast>` type. You'll want to move it down to the list starting at `%token END` below.